tests/dash/configs/privatelink_config.py

```diff
@@ -12,7 +12,8 @@
 VM1_CA = "10.0.0.11"  # VM customer address
 VM_CA_SUBNET = "10.0.0.0/16"
 PE_PA = "101.1.2.3"  # private endpoint physical address
-PE_CA = "10.2.0.100"  # private endpoint customer address
```

Comment:
Need to keep PE_CA around for the backward compability

Reply:
what is this used for?